src/pages/studyView/charts/barChart/BarChart.tsx

```diff
@@ -110,12 +111,20 @@ export default class BarChart extends React.Component<IBarChartProps, {}> implem
         return values;
     }
 
-    @computed get tickFormat() {
+    @computed get tickFormat() : string[] {
```

Comment:
This function does not always return an array of strings, its actual type is `(string|string[])[]`. See https://github.com/cBioPortal/cbioportal-frontend/blob/d635d95a416725a98f1ef1f628fe2c29a55ea88f/src/pages/studyView/charts/barChart/BarChart.tsx#L86